golang-github-osbuild-composer.spec

```diff
@@ -73,11 +75,17 @@ export GOFLAGS=-mod=vendor
 
 TEST_LDFLAGS="${LDFLAGS:-} -B 0x$(od -N 20 -An -tx1 -w100 /dev/urandom | tr -d ' ')"
```

Comment:
I think there should be `export`, otherwise this line has no effect.

Reply:
No, this works (and has been working on master), because this variable is only used locally in this script.

Reply:
Right, sorry, I got confused.